<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>girisPencere</class>
 <widget class="QWidget" name="girisPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Giriş Yap / Kayıt Ol - Music Library</string>
  </property>
  <widget class="QGroupBox" name="kayitolGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Kayıt Ol</string>
   </property>
   <widget class="QLineEdit" name="isim_input">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kaykadi_input">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kaysifre_input">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="kaykadi_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLabel" name="isim_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ad - Soyad</string>
    </property>
   </widget>
   <widget class="QLabel" name="kaysifre_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kayitolButon">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
   </widget>
   <widget class="QLabel" name="regInfo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="girisyapGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Giriş Yap</string>
   </property>
   <widget class="QLineEdit" name="kadi_input">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifre_input">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="kadi_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLabel" name="sifre_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="girisyapButon">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
   </widget>
   <widget class="QLabel" name="logInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
